main_test.go

```diff
@@ -64,6 +64,10 @@ func TestAcceptance(t *testing.T) {
 	if err := client.CompareAndSwap(last); err != nil {
 		t.Fatal(err)
 	}
+	large := &memcache.Item{Key: "large", Value: []byte("Hello World this text should be 128 bytes in size so that we may test the slab functionality of memcached. Filler text here to  ")}
```

Comment:
in my instance the slab number is 5. I don't know if this is due to the size of the text I added or what. Another pair of eyes or someone with more experience with memcache would be appreciated.